apps/io.vesea.funkysalamanders/manifest.json

```diff
@@ -0,0 +1,10 @@
+
+
+
```

Comment:
hi @Bread-VeSea , please remove the blank lines

Reply:
Done. Apologies for the oversight!